app/views/page/show.html.erb

```diff
@@ -232,6 +232,44 @@
             </div>
           <% end %>
 
+        <%# Publication %>
+        <% when 'PagePublicationComponent' %>
+          <div class="grid-container">
+            <% if @publication_ids.include?(component.id) %>
+              <%
+                component_index = @publication_list_component_index
+                plc = @publication_list_components[component_index]
+                @publication_list_component_index = component_index + 1
+              %>
+              <% if plc[:pagy].count <= 2 # use card styling for 2 or fewer  %>
+                <ul class="publication-component-list-cards usa-card-group dm-paginated-<%= component_index %>-events margin-bottom-4 <%= page_narrow_classes %><%= ' margin-bottom-0' if last_component === index %>">
+                  <%= render(partial: 'publications_list_cards', locals: {publications: plc[:publications]}) %>
+                </ul>
```

Comment:
2-card styling for VA Immersive homepage: https://www.figma.com/file/ccwLbt94U9QGfnuFCwOXzY/Core-Designs---VA-DM?type=design&node-id=6181%3A38719&mode=design&t=GsnXX0gGDh83hjz2-1